apps/src/templates/ArrowButtons.jsx

```diff
@@ -56,7 +56,7 @@ class ArrowButtons extends React.Component {
 
 const styles = {
   hidden: {display: 'none'},
-  visible: {display: 'inline-block'}
+  visible: {display: 'flex'}
```

Comment:
it doesn't make sense to me that these styles live with the react component, away from all the other styles -- we should make these classes, which can be applied conditionally. we may not even need the "visible" class, since flex is the default display for `#soft-buttons`

Reply:
I refactored this if you want to give it another look. It applies the "hidden" class or no class based on the `visible` property. 
[Visible example](http://localhost-studio.code.org:3000/s/hello-world-food-2021/lessons/1/levels/11)
[Hidden example](http://localhost-studio.code.org:3000/s/csd3-2021/lessons/7/levels/7)